### PR TITLE
Implement Archived Projects

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-629, L2B-846

Please review if the Archived Projects list is truthful

### CODE REVIEW

File: packages/config/src/layer2s/hermez.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ import { Layer2 } from './types'
 export const hermez: Layer2 = {
   type: 'layer2',
   id: ProjectId('hermez'),
+  isArchived: true,
```
Comment (luke7211) [inline packages/config/src/layer2s/hermez.ts:19]: I don't think Hermez is archived at this stage

Comment (luke7211) [reply packages/config/src/layer2s/hermez.ts:19]: No, Fuel is definitely not archived, it's up and running as we have experienced

File: packages/frontend/src/components/table/props/getBridgesRowProps.ts [hunk 2]
```diff
@@ -18,8 +19,10 @@ export function getBridgesRowProps(entry: BridgeTableEntry) {
     className: cx(
       getRowVerificationClassNames(entry),
       entry.type !== 'bridge' && 'hidden',
+      entry.isArchived && 'hidden opacity-50',
     ),
-    'data-combined-only': entry.type !== 'bridge' ? true : undefined,
+    'data-layer2': entry.type !== 'bridge' ? true : undefined,
```
Comment (antooni) [inline packages/frontend/src/components/table/props/getBridgesRowProps.ts:24]: A comment would be useful there

**Review by antooni — CHANGES_REQUESTED** [review]
Is this view on mobile desired?
image

File: packages/frontend/src/components/table/props/getBridgesRowProps.ts (hunk 2 above)
Comment (michalsidzej) [reply packages/frontend/src/components/table/props/getBridgesRowProps.ts]: Changed it, do you like it now?

**Review by antooni — APPROVED** [review]

**Pull request merged**